src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -300,6 +300,12 @@ public UInt160 GetCommitteeAddress(StoreView snapshot)
 
         private CachedCommittee GetCommitteeFromCache(StoreView snapshot)
         {
+            if (ProtocolSettings.Default.EmergencyMode.Contains(snapshot.Height))
+            {
+                // Restore original committee because the block it's in emergency mode
+                var candidates = GetCandidates(snapshot);
+                return new CachedCommittee(Blockchain.StandbyCommittee.Select(p => (p, candidates.FirstOrDefault(k => k.PublicKey.Equals(p)).Votes)));
```

Comment:
This will be a complete `HardFork`.

How will other nodes and the history sync?
We need to mark this on the Ledger somehow.

Reply:
A flag inside the block?

Reply:
Perhaps a flag will be enough, @shargon.
Let me talk to @igormcoelho about this situation first and discuss with him.